src/index.ts

```diff
@@ -107,14 +108,12 @@ async function main() {
           const CodaSDK = require("@o1labs/client-sdk");
```

Comment:
Why do we have this down here instead of up in the original import. It looks like it's part of the import on `main` (with an associated merge conflict) so want to make sure I'm not missing anything.

Reply:
It's just remnant. I think we should remove generate ephemeral entirely so didn't clean it -- I was going to do that after this.

Reply:
removed and updated pr